doc/source/user_guide/ug_running.rst

```diff
@@ -350,9 +350,19 @@ be modified.
 Porting
 -------
 
+There are four basis issues that need to be addressed when porting and these are addressed in four separate files in the script system,
+
+- setup of the environment such as compilers, environment variables, and other support software (in **env.[machine]_[environment]**)
+
+- setup of the Macros file to support the model build (in **Macros.[machine]_[environment]**)
+
+- setup of the batch scripts (in **cice.batch.csh**)
+
+- setup of the model launch (in **cice.launch.csh**)
```

Comment:
Thanks for that! I had noticed a few months ago that we did not mention `cice.launch.csh` and it was on my backlog to do a PR for that before the tutorial!